test/bin/mdx-dune-gen/misc/non-deterministic/dune

```diff
@@ -0,0 +1,52 @@
+(rule
+ (target dune_gen.ml)
+ (action
+  (with-stdout-to
+   %{target}
```

Comment:
you can just write `dune_gen.ml` here and remove `(target)`. then you can inline the `action`: `(rule (with-stdout-to dune-gen.ml (run ...)))`

Reply:
Ah, that's convenient. I've applied all these improvements and the file shrunk a lot. Thanks!

Reply:
np. I'm like clippy but for dune files.